percy-mirror-verifier/src/percy.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2022 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {request} from 'undici';
+
+export type PercySnapshot = {
+  type: 'snapshots';
+  id: string;
+  attributes: {
+    name: string;
+    'review-state':
+      | 'approved'
+      | 'unreviewed'
+      | 'changes_requested'
+      | 'failed'
+      | null;
+    'review-state-reason':
+      | 'no_diffs'
+      | 'auto_approved_branch'
+      | 'user_approved'
+      | null;
+  };
+};
+
+export type PercySnapshots = {
```

Comment:
nit suggestion (post-submit): `PercepSnapshotsResponse` might differentiate more clearly from the similarly-name singular `PercySnapshot` type and indicate that it's the Octokit response type

Reply:
Done in #1564